raft-tech/TANF-app
# Feat/1620 code signature

## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #1620 

## How to Test
Please note, this required repo-specific configurations including keys, gists, etc., and will not work on the HHS repo.

1. Assign any of the 'Deploy' labels to this PR or PR with this code and verify deploy action occurs.
1. Visit badge [location](https://github.com/raft-tech/TANF-app/blob/feat/1620-code-signature/docs/Technical-Documentation/TDP-environments-README.md#development)
1. Ensure branch and commit hash matches the environment selected in the Deploy label.
> *Demo GIF(s) and screenshots for testing procedure*
<img width="573" alt="Screenshot 2024-07-11 at 10 32 41 AM" src="https://github.com/raft-tech/TANF-app/assets/84722778/9b48bcaa-94a6-4a08-a1c8-65c80301796b">
<img width="1025" alt="Screenshot 2024-07-11 at 10 34 22 AM" src="https://github.com/raft-tech/TANF-app/assets/84722778/611ddc7d-2c7e-4768-a1e6-af189028a36c">

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x] We have a live representation or table showing the branch/commit and the Cloud.gov space.
+ [x] This table is accessible via a well-known link that is documented.

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
